selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOfParentOrganizationViewsTest.java

```diff
@@ -78,27 +71,27 @@ public void setUp() throws Exception {
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(childOrganization.getId());
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
-    testOrganizationServiceClient.deleteById(rootOrganization.getId());
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.